hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -160,6 +161,7 @@ private int doCluster(JavaSparkContext jsc) throws Exception {
     return this.doSchedule(jsc);
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
@chaplinthink Thanks for the code clean up, instead of adding annotation, would you please avoid the raw type usage firstly in the class?

Reply:
@leesf I think your suggestion is good, I have updated the code